### PR TITLE
Feat/analytics 1.19

### LINKED ISSUE #5334: Cleanup of analytics
Removing useless events, renaming and improving existing ones.

[Here](https://www.notion.so/satoshilabs/c63bdab01b704b099cab3bbb3227e1b1?v=406d107d6d584577a47d7580e8b2df89)

### LINKED ISSUE #5407: Analytics: 'select-wallet-type' always reports standard wallet type
**Describe the bug**
It works in 22.1.1 but it does not work in 22.2.3

```
  case SUITE.AUTH_DEVICE:
      analytics.report({
          type: 'select-wallet-type',
          payload: { type: action.payload.walletNumber ? 'hidden' : 'standard' },
      });
      break;
```

`walletNumber` is always `undefined` so wallet type is always `standard`

**Info:**

-   Suite Version: 22.5.2
-   Firmware: 1.11.1

**How to reproduce**

1. Open DevTools
2. Add hidden wallet
3. Check `select-wallet-type` analytics request
4. See that wallet type is `standard`

### PR DESCRIPTION
**Pull request by tomasklim**
close #5407
close #5334

- an anchor added to RouterLocationChange event
- many useless events removed
- some events got renamed, merged or improved
- added tracking of enabling/disabling coins
- fix reporting of standard/hidden wallets
- fix `Data Usage` switch state when app is not initialized. It was previously set to true. However, no data were collected, because analytics was not confirmed in onboarding

https://www.notion.so/satoshilabs/c63bdab01b704b099cab3bbb3227e1b1?v=406d107d6d584577a47d7580e8b2df89 instructions are here. However, I deleted removed events, so if you want to check, you need to use history.

### CODE REVIEW

**Comment by bosomt** [comment]
**QA OK**

**Info:**
 - Suite version: web 22.6.0 (fa84cf6e505a64df728bf37ce40fc1cdf9f5d709)
 - Browser: Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:100.0) Gecko/20100101 Firefox/100.0
 - OS: MacIntel
 - Screen: 1680x1050
 - Device: model T 2.5.1 regular (revision a1d22890ee6a9ffec4edfc6e1b5b6c74d23855a3)
 - Transport: bridge 2.0.31

**Comment by tomasklim** [comment]
@trezor/qa 
Together with fixed checkbox, it would be nice to test:

- `router/location-change` event should be sent when navigating in the app. If anchor is available, it should be part of this event. If transaction anchor is used, transaction id should not be send to analytics.
- `settings/device/wipe` event should be sent when wiping device
- `accounts/new-account` should be sent on creating new account
- `settings/coin/backend` should be send when changing coin backend
- `settings/coin` should be sent when enabling/disabling coin
- `settings/tor` should be sent when enabling/disabling tor (from settings, onboarding, coin backend)
- `settings/analytics` should be sent when enabling/disabling analytics (DISABLING is also important)
- `select-wallet-type` should be sent when choosing between standard and hidden wallet, and when adding new ones
- `menu/guide` is reported on opening guide (by bulb, modal, tooltip, F1)
- `menu/toggle-discreet` is reported when enabling/disabling discreet mode by menu icon/dashboard security section

Here are all analytics events and how to trigger them. If you want to explore analytics and check that everything is okay, you can go one by one https://www.notion.so/satoshilabs/c63bdab01b704b099cab3bbb3227e1b1?v=406d107d6d584577a47d7580e8b2df89

**Comment by bosomt** [comment]
**QA OK**

10x OK :)

- accounts/new-account 
- settings/device/wipe
- select-wallet-type 
- menu/guide
- menu/toggle-discreet
- accounts/new-account
- settings/coin
- settings/analytics
- router/location-change 
- settings/tor /tested in app/




**Info:**
 - Suite version: web 22.6.0 (03d7a0a4f843c478c087e5c24a8849542b898c46)
 - Browser: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/97.0.4692.71 Safari/537.36
 - OS: Linux x86_64
 - Screen: 1920x1080
 - Device: model 1 1.8.3 regular (revision df0963ec48f01f3d07ffca556e21ff0070cab099)
 - Transport: bridge 2.0.31

File: packages/suite/src/utils/suite/analytics.ts [hunk 1]
```diff
@@ -13,9 +13,31 @@ import {
...
+
+    return anchor.startsWith(AccountTransactionBaseAnchor) ? AccountTransactionBaseAnchor : anchor;
+};
+
+export const hasUserAllowedTracking = (
+    enabled: AnalyticsState['enabled'],
+    confirmed: AnalyticsState['confirmed'],
+) => {
+    const isConfirmed = !!confirmed;
+    const isEnabled = isConfirmed ? !!enabled : true;
```
Comment (matejkriz) [inline packages/suite/src/utils/suite/analytics.ts:35]: this ternary condition is not wrong, just useless - value returned from ` hasUserAllowedTracking` will be the same without it:
```suggestion
    const isEnabled = !!enabled;
```

**Review by matejkriz — APPROVED** [review]
👍 Small commits really helped during the review a lot ❤️

**Pull request merged**